services/database.js

```diff
@@ -66,30 +65,15 @@ function Database() {
 
     const connectionOptionsSequelize = { logging: false };
 
-    // NOTICE: mysql2 does not accepts unwanted options anymore.
-    //         See: https://github.com/sidorares/node-mysql2/pull/895
-    if (databaseDialect === 'mysql') {
-      // NOTICE: Add SSL options only if the user selected SSL mode.
-      if (isSSL) {
-        // TODO: Lumber should accept certificate file (CRT) to work with SSL.
-        //       Since it requires to review onboarding, it is not implemented yet.
-        //       See: https://www.npmjs.com/package/mysql#ssl-options
-        connectionOptionsSequelize.dialectOptions = {
-          ssl: { rejectUnauthorized: isSSL },
-        };
-      }
-    } else if (databaseDialect === 'mssql') {
-      connectionOptionsSequelize.dialectOptions = {
-        options: {
-          encrypt: isSSL,
-        },
-      };
-    } else {
-      connectionOptionsSequelize.dialectOptions = {
-        ssl: isSSL,
-      };
+    if (databaseDialect === 'mssql') {
+      connectionOptionsSequelize.dialectOptions = { options: { encrypt: isSSL } };
+    } else if (isSSL) {
+      // Add SSL options only if the user selected SSL mode.
+      // SSL Cerificate is always trusted during `lumber generate` command to ease their onboarding.
+      connectionOptionsSequelize.dialectOptions = { ssl: { rejectUnauthorized: false } };
```

Comment:
[Question] Shouldn't this be done also for `mssql` ? I'm not aware of this being an issue, but just want to be sure

Reply:
`mssql` was already considered different and doesn't have the same options. However, the default behavior is already to trust certificates. Source: http://tediousjs.github.io/tedious/api-connection.html

> options.trustServerCertificate
If "true", the SQL Server SSL certificate is automatically trusted when the communication layer is encrypted using SSL. If "false", the SQL Server validates the server SSL certificate. If the server certificate validation fails, the driver raises an error and terminates the connection. Make sure the value passed to serverName exactly matches the Common Name (CN) or DNS name in the Subject Alternate Name in the server certificate for an SSL connection to succeed. (default: true).